bench/simple_consumer_bench.py

```diff
@@ -0,0 +1,111 @@
+import json
```

Comment:
Thanks, yes, will add documentation! These really got written in anger, so I was half thinking removing them would be cleaner, but on the other hand... we may want to benchmark again when making big or small changes that could affect performance. Keep?